src/reducers/notifications.ts

```diff
@@ -234,31 +234,35 @@ export const notificationUpdater: Middleware =
     }
 
     return (action: any) => {
-      const state = getState() as any as IRootState;
-      const network = Util.networkName(state.web3.networkId).toLowerCase();
-
-      if (action.type === REHYDRATE) {
-        const a = action as RehydrateAction;
-        if (a.payload) {
-          const operations = a.payload.operations as IOperationsState;
-          Object.keys(operations).forEach((id) => {
-            if (operations[id].status === OperationStatus.Sent || operations[id].error) {
-              transaction2Notification(network, id, operations[id]);
-            }
-          })
+
+      (async () => {
+
+        const state = getState() as any as IRootState;
+        const network = Util.networkName(Number(await Arc.Utils.getNetworkId())).toLowerCase();
```

Comment:
You can call `(await Arc.Utils.getNetworkName()).toLowerCase()`

Reply:
Thanks! fixed: https://github.com/daostack/alchemy/pull/152/commits/836ef7bfb6a1f3039bd1319d5795479c9f7fc79c